src/developer-quickstart.md

```diff
@@ -147,8 +147,7 @@ impl Counter for Contract {
 }
 ```
 
-> **Note**
-> `return storage.counter;` is equivalent to `storage.counter`.
+> **Note** > `return storage.counter;` is equivalent to `storage.counter`.
```

Comment:
The notes in this file don't all look the same. No strong preference for one form or another, but they should probably be standardized? I personally like `**Note**: ` and keeping the whole note on one line.